src/main/java/com/laborete/LaboreteAPI/repository/UsersRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.laborete.LaboreteAPI.repository;
+
+import com.laborete.LaboreteAPI.entity.Users;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.UUID;
+
+@Repository
+public interface UsersRepository extends JpaRepository<Users, Long> {
```

Comment:
The second param in the generic of `JpaRepository` is a type of id. So, it should be `UUID` instead of `Long` in your case.